tests/integration/jersey-4697/src/test/java/org/glassfish/jersey/tests/integration/jersey4697/MonitoringEventListenerTest.java

```diff
@@ -16,13 +16,17 @@
 
 package org.glassfish.jersey.tests.integration.jersey4697;
 
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.junit.Assert.assertEquals;
```

Comment:
In the latest version of the PR there are many redundant imports. Could you please clean them out? 

And those imports, which are already in place (before any changes) should not be reformatted because when it comes to merges, reformatting brings unnecessary conflicts.

Reply:
Good point